front-end/src/ui-components/HelperTooltip.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Icon, Popup } from 'semantic-ui-react';
+
+interface Props {
+	className?: string
+	content: string
+}
+
+const HelperTooltip = ({ className, content }:Props) =>
+	<Popup
+		className={className}
+		trigger={<Icon name='question circle' style={{ color: '#A6ACB3', marginLeft: '0.2rem' }}/>}
```

Comment:
You can style this icon outside, in a separate const, and just reference it in the trigger.